code/go/internal/validator/test/packages/good/data_stream/ilm_policy/elasticsearch/ilm/diagnostics.json

```diff
@@ -0,0 +1,15 @@
+{
+    "policy": {
```

Comment:
I wonder if we can provide the schema for this file. I don't know if Elasticsearch team plans to extend this file in the future or if it's rather never-changing, concise schema.

We haven't defined the schema for ingest pipelines, so I'm to leave it as it (without schema definition for ILM policy).